pypeclub/OpenPype
# Hiero: loaded clip was not set colorspace from version data

# testing notes
- open hiero at your testing timeline
- set one of clip for publishing to some crazy colorspace like `cineon` or `Alexa`
- publish the clip 
- open Loader and load the subset back to your timeline
- double click to the clip on timeline > properties bin should open
- look for `colorspace` attribute and see if it is the same as the one you had set previously
- result should be the same.
